docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,171 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on three Kubernetes clusters -- a Minikube cluster, an on-premise CPU
```

Comment:
on three Kubernetes clusters:

1. a Minikube cluster,
1. an on-premise CPU cluster managed by the tailored Kubernetes system, and
1. a GPU cluster managed by the same tailored system.